gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notifications-list.component.ts

```diff
@@ -113,4 +126,39 @@ export class NotificationsListComponent implements OnInit {
       )
       .subscribe(() => this.ngOnInit());
   }
+
+  addNotification() {
+    this.matDialog
+      .open<NotificationsAddNotificationDialogComponent, NotificationsAddNotificationDialogData, NewNotification>(
+        NotificationsAddNotificationDialogComponent,
+        {
+          width: '500px',
+          data: {
+            notifier: this.notifiersGroup,
+          },
+          role: 'dialog',
+          id: 'addNotificationDialog',
+        },
+      )
+      .afterClosed()
+      .pipe(
+        filter((result) => !!result),
+        switchMap((newNotification) =>
+          this.notificationSettingsService.create(this.ajsStateParams.apiId, this.addNotificationPayloadBuilder(newNotification)),
+        ),
+        tap(() => {
+          this.snackBarService.success('Notification created successfully');
+        }),
+        catchError(({ error }) => {
+          this.snackBarService.error(error.message);
+          return EMPTY;
+        }),
+        takeUntil(this.unsubscribe$),
+      )
+      .subscribe(() => this.ngOnInit());
+  }
+
+  addNotificationPayloadBuilder(newNotification: NewNotification) {
```

Comment:
i think you need to remove this and complete le NewNotification ts type. and make the dialog return a complete NewNotification obj. WDYT ?

Reply:
Done